src/tests/ftest/control/daos_agent_support_collect_log.py

```diff
@@ -0,0 +1,51 @@
+"""
+  (C) Copyright 2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from support_test_base import SupportTestBase
+
+
+class DaosAgentSupportCollectLogTest(SupportTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test Class Description:
+            Verify the daos_server support collect-log command.
+
+    :avocado: recursive
+    """
+
+    def test_daos_agent_support_collect_log(self):
+        """JIRA ID: DAOS-10625
+
+        Test Description:
+            Test daos_agent support collect-log command completes successfully.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,medium
```

Comment:
I wonder if these tests actually need hw to pass...

Anyway, with google/2.4 we explicitely disable hardware testing but for a one off run of these tests, you can do this

Test-tag: support pr
Skip-func-hw-test-medium: false

in the last commit.  If you want to do this but not making any changes, I suggest

git commit --allow-empty

Also the DCO check wants commits to have Signed-off-by so git commit -s will do that.